test/ode/ode_dense_tests.jl

```diff
@@ -64,6 +64,24 @@ sol2 =solve(prob,Midpoint(),dt=1//2^(4),dense=true)
 
 prob = prob_ode_linear
 
+sol =solve(prob,SSPRK22(),dt=1//2^(2),dense=true)
+
+sol(interpd_1d,0:1//2^(4):1)
+
+sol2 =solve(prob,SSPRK22(),dt=1//2^(4),dense=true)
+
+@test maximum(map((x)->maximum(abs.(x)),sol2[:] - interpd_1d)) < 0.15
```

Comment:
If you drag and drop a file into the textbox then Github handles it.

![bifplot](https://cloud.githubusercontent.com/assets/1814174/24711213/26faaf9e-19d4-11e7-90c6-9c05c446786d.png)